libraries/c-sharp/RiotWrapper.NET

```diff
@@ -0,0 +1,17 @@
+{
+    "owner": "ChristianFreak, Chris/PJC",
```

Comment:
Hey Chris!

We use this field to get details about the Github repo, so could you just change this to `ChristianFreak`, since that's the account name that owns the repo? 

Thanks!